### DISCUSSION TITLE
How Far Back Does OP3 Keep Data From?

### DISCUSSION THREAD

**Discussion by SpencerWF** [skymethod/op3#26] [Q&A]
It seems like the downloads data for our podcast is only kept 3 months back.  Is this accurate, or have I missed something?

**Comment by johnspurlock-skymethod** [comment]
It has data as far back as when you initially added the prefix, are you using the [api](https://op3.dev/api/docs)?

**Reply by jamescridland** [reply]
It doesn’t in the UI - it looks like it only goes back a few months. What are the chances of that being added?

**Reply by johnspurlock-skymethod** [reply]
Chances are good of that being added if the sales team can find a new monthly sponsor being added!

<img width="161" alt="image" src="https://github.com/skymethod/op3/assets/47259736/8c66928e-77e9-424d-ae22-970eed11a4f4">
